<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-C55NDTU" failures="3" tests="4" name="Test2" time="81.284" errors="0" timestamp="2025-05-23T04:17:46 PDT">
  <testcase classname="paralleltestng.ParallelTestng" name="PIM" time="2.79">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index 1 out of bounds for length 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index 1 out of bounds for length 0
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:100)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:106)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:302)
at java.base/java.util.Objects.checkIndex(Objects.java:365)
at java.base/java.util.ArrayList.get(ArrayList.java:428)
at paralleltestng.ParallelTestng.PIM(ParallelTestng.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- PIM -->
  <testcase classname="paralleltestng.ParallelTestng" name="admin" time="0.029">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index 0 out of bounds for length 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:100)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:106)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:302)
at java.base/java.util.Objects.checkIndex(Objects.java:365)
at java.base/java.util.ArrayList.get(ArrayList.java:428)
at paralleltestng.ParallelTestng.admin(ParallelTestng.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- admin -->
  <testcase classname="paralleltestng.ParallelTestng" name="leave" time="0.021">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index 2 out of bounds for length 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index 2 out of bounds for length 0
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:100)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:106)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:302)
at java.base/java.util.Objects.checkIndex(Objects.java:365)
at java.base/java.util.ArrayList.get(ArrayList.java:428)
at paralleltestng.ParallelTestng.leave(ParallelTestng.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- leave -->
  <testcase classname="paralleltestng.ParallelTestng" name="login" time="8.049"/>
</testsuite> <!-- Test2 -->
